Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -61,9 +118,20 @@ public struct FlyoverSceneView: View {
                         orientation: interfaceOrientation
                     )
                 }
+                .onChange(of: initialCamera) { initialCamera in
+                    cameraController.originCamera = initialCamera
+                }
+                .onChange(of: translationFactor) { translationFactor in
+                    cameraController.translationFactor = translationFactor
+                }
```

Comment:
@des12437 can you look at if we need similar logic for the tabletop scene view? You can try adding a button that updates the anchor point and/or translation factor and see if the changes are propagated. I assume they won't be and similar logic will be required.